docs/.snippets/code/developer-tools/chopsticks/overview/basic-example.js

```diff
@@ -0,0 +1,12 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
+
+async function connectToFork() {
+  const wsProvider = new WsProvider("ws://localhost:8000");
```

Comment:
This should be formatted with single quotes, other than that I think it looks ok

Reply:
I switched them out for single quotes. Then when I highlight the code block and "Format Selection with Prettier" in my VSCode, it puts them back to double quotes. So I need to know which one is correct, please.

Reply:
Did you edit your prettier config to align with the docs standards in the style guide? Seems like thats the problem, because they should be single quotes, trailing commas, 80 char line max, etc.